### DISCUSSION TITLE
Virtual Camera Resolution Lower Than Main Display

### DISCUSSION THREAD

**Discussion by CalvinKroese** [lucasw/rviz_camera_stream#41] [Q&A] [answered]
When creating multiple virtual camera displays in RVIZ it starts becoming noticeable that the render resolution is lower than the main 3D display. 

I am setting the image width and height to 1920 x 1080, but this does not seem to alter the render window, changing the resolution.

Is this a known issue, and is there a way around it?

**Comment by lucasw** [comment, marked as answer]
I edited the `demo.launch` to get a 1920x1080 image:

```
diff --git a/launch/demo.launch b/launch/demo.launch
index f9ea44c..95ef1c3 100644
--- a/launch/demo.launch
+++ b/launch/demo.launch
@@ -29,13 +29,13 @@
   <node name="camera_info" pkg="rostopic" type="rostopic"
       args="pub camera_info sensor_msgs/CameraInfo
      '{header: {seq: 0, stamp: {secs: 0, nsecs: 0}, frame_id: 'camera2'},
-      height: 720, width: 1280, distortion_model: 'plumb_bob',
+      height: 1080, width: 1920, distortion_model: 'plumb_bob',
       D: [0],
       K: [300.0, 0.0, 640, 0.0, 300.0, 360.0, 0.0, 0.0, 1.0],
       R: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0],
       P: [300.0, 0.0, 640, 0.0, 0.0, 300, 360, 0.0, 0.0, 0.0, 1.0, 0.0],
       binning_x: 0, binning_y: 0,
-      roi: {x_offset: 0, y_offset: 0, height: 720, width: 1280, do_rectify: false}}' -r 2"
+      roi: {x_offset: 0, y_offset: 0, height: 1080, width: 1920, do_rectify: false}}' -r 2"
       output="screen"/>
   </group>
```

(not sure about the roi but made those edits anyhow)

I opened up the output image topic in rqt_image_view, and then save an image to disk:

![rviz_camera_stream_1920_1080](https://user-images.githubusercontent.com/1334122/139593606-6b95ded5-2eb3-4853-8057-d0d613975e37.png)

It looks like I'm getting all the pixels I asked for when zooming into that image:
![rviz_camera_stream_gimp_closeup_pixels](https://user-images.githubusercontent.com/1334122/139593744-5f939da4-aea7-436b-ba1c-1f4821c38141.png)

Does that turn out differently for you?

**Reply by CalvinKroese** [reply]
Thanks for the reply, I also went about checking it in the same fashion which proved to be useful.
